attrici/commands/detrend.py

```diff
@@ -93,19 +86,22 @@ def add_parser(subparsers):
     group.add_argument(
         "--modes",
         type=int,
-        default=4,
+        default=Config.__dataclass_fields__["modes"].default,
```

Comment:
Is this better than hardcoded values for tests? A user may change them and break tests by accident.

Reply:
Not sure I understand correctly, but the change to use `Config.__dataclass_fields__["modes"].default` is to read the default from the 'Config' object in `detrend.py` as a single source of truth for the default.

Sorry, we should have put this more clearly in the PR description.